content/en/docs/architecture/step-registry.md

```diff
@@ -462,7 +462,8 @@ fields:
 
 * `name`: environment variable name
 * `default` (optional): the value assigned if none is provided
-* `documentation` (optional): a textual description of the parameter
+* `documentation` (optional): a textual description of the parameter. **This field is markdown syntax**, 
```

Comment:
I think using some Markdown in the examples makes sense, but I dunno if we need the `**` here.  Otherwise looks good to me, and I'll survive if folks want to keep the `**` too ;)

Reply:
Maybe just add "Markdown supported". That should be good enough

Reply:
Not sure. The term "supported" suggests that it's optional. But this field is now always in markdown synyax. For ecample, I used '<' in my doc, snd iy was escaped.

Reply:
For common usage, I think that is fine.

Reply:
Thanks @smg247 and @wking - done.